server/cmwell-ws/app/actions/ActiveInfotonGenerator.scala

```diff
@@ -603,7 +603,9 @@ ${lines.mkString("\n")}
   import scala.language.implicitConversions
 
 
-  def generateInfoton(host: String, path: String, md: DateTime = new DateTime(), length: Int = 0, offset: Int = 0, isRoot : Boolean = false, nbg: Boolean = false, fieldFilters: Option[FieldFilter]): Future[Option[VirtualInfoton]] = {
+  def generateInfoton(host: String, path: String, now: Long, length: Int = 0, offset: Int = 0, isRoot : Boolean = false, nbg: Boolean = false, fieldFilters: Option[FieldFilter]): Future[Option[VirtualInfoton]] = {
+
+    val md: DateTime = new DateTime(now)
```

Comment:
rename md => d // this confuses me, md can be modifiedDate as well as MarkDown :)